components/camel-kafka/src/main/java/org/apache/camel/component/kafka/KafkaConfiguration.java

```diff
@@ -775,6 +779,20 @@ public void setBrokers(String brokers) {
         this.brokers = brokers;
     }
 
+    public String getSchemaRegistryURL() {
+		return schemaRegistryURL;
+	}
+
+    /**
+     * URL of the Kafka schema registry to use.
+     * The format is host1:port1,host2:port2.
+     * <p/>
+     * This option is known as <tt>schema.registry.url</tt> in the Kafka documentation.
+     */
```

Comment:
I think it must be stressed that this something related to Confluent only. There is no trace of schema.registry.url in the official Kafka documentation. So I think the javadoc must be changed. Also you need to run a build of the component so the associated .adoc file will be aligned to the new parameter.

Reply:
Yeah that must be documented that its confluent only.